scripts/clean_jobs.py

```diff
@@ -0,0 +1,92 @@
+# Read in jobs from the _data/jobs.yml file, file links that are both
+# expired and not working, and remove them. Write to a new file.
+# Copyright @vsoch, 2020
+
+import os
+import datetime
+from urlchecker.core.urlproc import check_urls
+import shutil
+import sys
+import tempfile
+import yaml
+
+here = os.path.dirname(os.path.abspath(__file__))
+
+
+def get_filepath():
+    """load the jobs file.
+    """
+    filepath = os.path.join(os.path.dirname(here), "_data", "jobs.yml")
+
+    # Exit on error if we cannot find file
+    if not os.path.exists(filepath):
+        print("Cannot find %s" % filepath)
+
+    return filepath
+
+
+def read_jobs(filepath):
+    """read in the jobs data.
+    """
+    # Read in the entire membership counts
```

Comment:
membership counts?

Reply:
You got me! I was adopting the previous function from update membership counts. Fixing right away!